src/test/cli/crushtool/add-item.t

```diff
@@ -1,5 +1,7 @@
   $ crushtool -i "$TESTDIR/simple.template" --add-item 0 1.0 device0 --loc host host0 --loc cluster cluster0 -o one > /dev/null
   $ crushtool -i one --add-item 1 1.0 device1 --loc host host0 --loc cluster cluster0 -o two > /dev/null
+  $ crushtool -i two --create-simple-rule simple-rule cluster0 host firstn -o two > /dev/null
```

Comment:
can you crushtool -d ... after each of these new commands so that the we can see the rule (or lack of the rule) in the dump to confirm it is actually working?